src/pranitaPJun20/AllPrimeNumber.java

```diff
@@ -0,0 +1,64 @@
+/*Assignment 19 : 27th Jun'2020
+
+Program 1: Create a class to satisfy below requirements. 
+                  a) Given number is prime or not. 
+                  b) Print total count of prime numbers between 2->100.
+                  c) Find sum of all prime numbers between 2->100
+                  d) Find avg of prime numbers between 2->100
+*/
+
+package pranitaPJun20;
+
+public class AllPrimeNumber {
+//Method to check Given number is prime or not. 
+	void checkPrime(int num) {
+		boolean flag=true;
+		for(int index=2;index<=num/2;num++) {
+			if(num%index==0) {
+				System.out.println("Given number "+num+" is not prime");
+				flag=false;
+				break;
+			}
+		}
+		if(flag==true) {
+			System.out.println("Given number "+num+" is prime");
+	
+		}	
+	}
+	
+//Method to check Count,Sum and average of all prime numbers in range
+	boolean checkPrimeNumbRange(int num) {
+		boolean flag=true;
+		for(int index=2;index<=num/2;index++) {
+			if(num%index==0) {
+				flag=false;
+				break;
+				
+			}
+		}
+		return flag;
+	}
+	
+	public static void main(String[] args) {
+		AllPrimeNumber allPrimeNumber = new AllPrimeNumber();
+		int num =15;
+		allPrimeNumber.checkPrime(num);
+		int sum=0;
+		int count=0;
+		for(int num1=2;num1<=100;num1++) {
```

Comment:
Clearly, said that you are not going to write any logic in the main method then ???

Reply:
Written logic in method now.